### PR TITLE
Remove a sentence from the disability wizard

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Changes the content in the disability compensation wizard per the connected ticket.

## Testing done
Manually tested.
Updated unit test.

## Screenshots
image

## Acceptance criteria
- [x] The sentence in question is removed

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by markgreenburg — APPROVED** [review]
lgtm

**Comment by cvalarida** [comment]
@peggygannon Can you take a quick look at this one? I meant to add you as a reviewer when I made the PR. :man_facepalming:

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cvalarida per the issue for this, the sentence should now read. Can we make that update?

To file your first disability claim, please go to our eBenefits website.

**Comment by cvalarida** [comment]
Updated button text:
image

**Comment by cvalarida** [comment]
Updated content:
image

File: src/applications/disability-benefits/526EZ/components/ButtonContainer.jsx [hunk 1]
```diff
@@ -24,7 +23,7 @@ export default function ButtonContainer(props) {
         </button>
       )}
       {atIncreaseGuidance &&
-        !conditionalStorage().getItem('userToken') && (
+        !isLoggedIn && (
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/components/ButtonContainer.jsx]: Oh that's nice

**Review by peggygannon — APPROVED** [review]

**Pull request merged**